src/common/enums/user.ts

```diff
@@ -16,3 +18,6 @@ export const AUTHOR_TYPE = {
   default: 'default',
   trendy: 'trendy',
 } as const
+
+export const PUBLISH_ARTICLE_RATE_LIMIT = isProd ? 1 : 1000
```

Comment:
is this rate confirmed with the CC team?

Reply:
it was the default before:

"hotfix(publish-rate): reduce rate of publishArticle"

This reverts commit 2717c88.